### DISCUSSION TITLE
PrivateGPT in other languages

### DISCUSSION THREAD

**Discussion by apendo-emil** [imartinez/privateGPT#446] [Q&A]
Hi!

How do you use PrivateGPT in other languages than english? I see some threads below, but I don't understand how to achieve this. Can someone supply a ste-by-step instruction?

**Comment by amroeser** [comment]
Hey have a look at the topic #221, here I have deposited a code for German that works

**Reply by apendo-emil** [reply]
I'll have a look. Thanks for the help.
